### DISCUSSION TITLE
yarn config set npmAuthToken for npmRegistries command fails on windows

### DISCUSSION THREAD

**Discussion by articot** [yarnpkg/berry#4119] [Q&A]
yarn berry v3.1.1

there is a documentation page:

https://yarnpkg.com/cli/config/set

and there is a  sample command 

```yarn config set 'npmRegistries["//npm.example.com"].npmAuthToken' "ffffffff-ffff-ffff-ffff-ffffffffffff"```

and it runs well under linux based os

but fails under Windows 10

![image](https://user-images.githubusercontent.com/52530805/154022258-d82ddb29-2bbc-4fac-8c9a-3ad49c25c2e3.png)

it results in the following in .yarnrc.yml

![image](https://user-images.githubusercontent.com/52530805/154022401-85d3b96b-2ece-471c-afd9-66ffd0e6b499.png)

**Comment by viceice** [comment]
- #4881
